<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>166</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="piece00">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 1px 1px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece01_3">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 1px 1px 0px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece02_3">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 1px 1px 0px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece03_3">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 1px 1px 0px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece04_3">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 0px 1px 0px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece05">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 1px 1px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="piece10">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px 1px 1px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="piece20">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px 1px 1px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="piece20_2">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px 1px 1px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="piece10_2">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px 1px 0px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: white;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: white;</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="home0">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 1px solid black;
background-color: transparent;</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: white;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="home2">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 1px solid black;
background-color: transparent;</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: white;</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="home1">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 1px solid black;
background-color: transparent;</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: white;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="home3">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 1px solid black;
background-color: transparent;</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: white;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: white;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="piece15">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px 1px 1px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="piece25">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px 1px 1px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="piece25_2">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px 1px 1px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="piece15_2">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px 1px 0px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="piece00_2">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 1px 1px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece01_2">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 1px 1px 0px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece02_2">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 1px 1px 0px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece03_2">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 1px 1px 0px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece04_2">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 0px 1px 0px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece05_2">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px 1px 1px 1px;
border-style: solid;
border-color: black;
background-color: transparent;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
